website/content/docs/ecs/upgrade-to-dataplanes.mdx

```diff
@@ -0,0 +1,69 @@
+---
+layout: docs
+page_title: Upgrade to Consul dataplane architecture
+description: Learn how to upgrade your existing Consul service mesh on ECS workloads to the agentless dataplanes architecture.
+---
+
+# Upgrade to Consul dataplane architecture
+
+This topic describes how to manually upgrade a live installation of Consul on ECS to the dataplane-based architecture with zero downtime. Since v1.0.0, Consul service mesh on ECS uses [Consul dataplanes](/consul/docs/connect/dataplane), which are lightweight processes for managing Envoy proxies in containerized networks. Refer to the [release notes](/consul/docs/release-notes/consul-ecs/v1_0_x) for additional information about the switch to Consul dataplanes.
```

Comment:
```suggestion
This topic describes how to manually upgrade a live installation of Consul on ECS to the dataplane-based architecture with zero downtime. Since v0.7.0, Consul service mesh on ECS uses [Consul dataplanes](/consul/docs/connect/dataplane), which are lightweight processes for managing Envoy proxies in containerized networks. Refer to the [release notes](/consul/docs/release-notes/consul-ecs/v0_7_x) for additional information about the switch to Consul dataplanes.
```